api/DOMMatrixReadOnly.json

```
@@ -1306,12 +1306,9 @@
             "edge": "mirror",
             "firefox": {
               "version_added": "33",
-              "notes": "Firefox 69 introduced support for the modern six-parameter syntax for <code>scale()</code>. Previously, it only supported the older three-parameter syntax: <code>scale(<em>scaleX</em>[, <em>originX</em>][, <em>originY</em>]]])</code>."
-            },
-            "firefox_android": {
-              "version_added": "33",
-              "notes": "Firefox for Android 79 introduced support for the modern six-parameter syntax for <code>scale()</code>. Previously, it only supported the older three-parameter syntax: <code>scale(<em>scaleX</em>[, <em>originX</em>][, <em>originY</em>]]])</code>."
+              "notes": "Version 69 introduced support for the modern six-parameter syntax for <code>scale()</code>. Previously, it only supported the older three-parameter syntax: <code>scale(scaleX[, originX][, originY]]])</code>."
```

Shouldn't this be "Firefox", rather than "Version", so that https://github.com/mdn/browser-compat-data/pull/24910 can properly replace "Firefox" with "Firefox for Android"?

```suggestion
              "notes": "Firefox 69 introduced support for the modern six-parameter syntax for <code>scale()</code>. Previously, it only supported the older three-parameter syntax: <code>scale(scaleX[, originX][, originY]]])</code>."
```

---

I would think that "Firefox 69" is better than "Version 69", because it is more explicit, and makes it harder to accidentally copy a note from say a Firefox statement to a Chrome statement (which happens, cf. https://github.com/mdn/browser-compat-data/pull/24830).

Counts:

```
% rg 'Version \d+ ' | wc -l
      26
% rg 'Firefox \d+ ' | wc -l
      82
```